apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -30,22 +30,56 @@ fun <T> Expect<T>.notToBe(expected: T): Expect<T> = _logicAppend { notToBe(expec
 /**
  * Expects that the subject of the assertion is the same instance as [expected].
  *
+ * Deprecated as atrium moves to a consistent ‘to + infinitive’ naming convention. Use [toBeTheSameAs] instead.
+ * This function will be removed in version 1.0.0. See
+ * [atrium-roadmap#93](https://github.com/robstoll/atrium-roadmap/issues/93) for details and to give feedback.
+ *
  * @return An [Expect] for the current subject of the assertion.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+@Deprecated(
+    "Replaced by toBeTheSameAs. Will be removed with version 1.0.0",
+    ReplaceWith("this.toBeTheSameAs<T>(expected)")
+) // TODO remove with 1.0.0
+fun <T> Expect<T>.isSameAs(expected: T): Expect<T> = toBeTheSameAs(expected)
+
+/**
+ * Expects that the subject of `this` assertion is the same instance as [expected].
```

Comment:
I am not entirely sure if `this` helps or confuses because by only reading this sentence, I would guess this function is an extension of `Assertion` due to the way `this` is wrapped with back-ticks. 
However, I think it would be an improvement if it is not wrapped with back-ticks and could be changed everywhere.

Reply:
I usually like writing ‘this’ as `this`, to point out that we are talking about the programming meaning of ‘this’. However, I agree that it is a little irritating because the receiver type is `Expect`.

Maybe this works:

 > Expects that the subject of `this` expectation is the same instance as [expected].

But I am also fine with just writing ‘this’, without the backticks. You’ll be the judge.

Reply:
> Expects that the subject of `this` expectation is the same instance as [expected].

Sound good, we should replace `subject of the assertion` in all places with ``subject of  `this` expectation``

Reply:
:+1:. I think we should try to adopt the lingo ‘expectation’ instead of ‘assertion’ everywhere.

Reply:
Agreed. Let's wait with such changes until we merge into master to avoid conflicts. I create a project, where we can track such tasks

Reply:
I’d suggest the opposite: Do these simple improvements now, because they are the smaller, easier changes, and start the renaming afterwards. What do you think?

Reply:
Good as well if you create a PR targeting master, then we can do it before merging the renames into master. But we should wait until I merge the MapLike.contains as we need to change those files as well.

Reply:
going to address this now.